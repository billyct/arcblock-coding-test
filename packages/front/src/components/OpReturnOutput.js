const OpReturnOutput = () => (
  <div className='uppercase py-2 '>
    OP_RETURN
  </div>
)

export default OpReturnOutput
