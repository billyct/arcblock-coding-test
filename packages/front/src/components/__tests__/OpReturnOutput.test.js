import {render} from '@testing-library/react'

import OpReturnOutput from '../OpReturnOutput'

test('OpReturnOutput Component', () => {
  const {container} = render(<OpReturnOutput/>)
  expect(container).toMatchSnapshot()
})
