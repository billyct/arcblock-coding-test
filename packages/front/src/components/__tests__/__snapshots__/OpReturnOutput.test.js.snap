// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpReturnOutput Component 1`] = `
<div>
  <div
    class="uppercase py-2 "
  >
    OP_RETURN
  </div>
</div>
`;
